corehq/apps/accounting/migrations/0077_add_web_user_feature.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations
+from django.core.management import call_command
+
+from corehq.privileges import LOADTEST_USERS
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.apps.accounting.bootstrap.utils import _ensure_feature_rates
+from corehq.apps.accounting.bootstrap.config.web_user_feature_rate import BOOTSTRAP_CONFIG
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        LOADTEST_USERS,
+        skip_edition='Paused,Community',
+        noinput=True,
+    )
+
+
+def _add_web_user_feature(apps, schema_editor):
+    Feature = apps.get_model('accounting', 'Feature')
+    web_user_feature = Feature.objects.create(name='Web User', feature_type='Web User')
```

Comment:
Do you think using `corehq.apps.accounting.models.FeatureType.WEB_USER` instead of `Web User` string can work here?

Reply:
Yes, I think that would work and would be better. Fixed here: ca46f909b5614fed07b4842f0be91aaf6eef714b